corehq/apps/sms/signals.py

```diff
@@ -0,0 +1,17 @@
+from casexml.apps.case.signals import case_post_save
+from casexml.apps.case.models import CommCareCase
+from dimagi.utils.logging import notify_exception
+
+
+def case_changed_receiver(sender, case, **kwargs):
+    try:
+        from corehq.apps.sms.tasks import sync_case_phone_number
+        sync_case_phone_number.delay(case)
```

Comment:
another option could be to make this a pillow, but not sure that that's better. so think you're right that this is probably fine